api/PerformanceResourceTiming.json

```
@@ -804,7 +804,7 @@
           "spec_url": "https://w3c.github.io/resource-timing/#dom-performanceresourcetiming-workerstart",
           "support": {
             "chrome": {
-              "version_added": "43"
+              "version_added": "46"
```

https://bugzilla.mozilla.org/show_bug.cgi?id=1191943 says 58.
```suggestion
              "version_added": "58"
```